### ISSUE TITLE
Please add remove_all_emoji util method for cleaning texts.

### ISSUE THREAD

**Issue by dimparf** [veelenga/emoji.cr#4]

**Comment by veelenga** [comment]
Good idea. Will take care of this. Spec:

```crystal
Emoji.sanitize(":football: is awesome") #=> " is awesome"
```

**Comment by dimparf** [comment]
@veelenga I would like to use your library for the cleaning of unnecessary text characters (word processing for machine learning). This method will be very useful for me.

**Comment by veelenga** [comment]
@dimparf that method is going to clean only emoji, like `:football:` in the example. If you need to clean other chars/words, unfortunately you have to do your custom sanitization that matches your needs.

**Comment by veelenga** [comment]
@dimparf i guess you want to clean already processed emoji:

```crystal
Emoji.sanitize("⚽  is awesome") #=> " is awesome"
```

⚽  is represented by the codepoint `U+26BD`, not by the string `":soccer:"`. Probably this is also possible, but i am not sure if it will work in all cases.

**Comment by dimparf** [comment]
Yes, i want to clean already processed emoji.

**Comment by veelenga** [comment]
Closed by 0768f61
